app/controllers/evms_controller.rb

```diff
@@ -60,6 +60,15 @@ def index
     end
     @no_data = issues.blank?
     @no_data_incomplete_issues = @incomplete_issues.blank?
+    # export
+    respond_to do |format|
+      format.html
+      format.csv {
+        send_data(@project_evm.to_csv,
+                  :type => 'text/csv; header=present',
+                  :filename => 'evm_' + @project.name + '_' +Time.now.to_date.to_s + '.csv')
```

Comment:
Line is too long. [92/80]<br>Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `+`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.